### DISCUSSION TITLE
why the packages workflow for the CI instead the bottle style releases using test bot?

### DISCUSSION THREAD

**Discussion by ipatch** [davidchall/homebrew-hep#237] [General]
hello 👋

as one of the maintainers for the homebrew-freecad tap i'm curious as to why you decided to build the releases using the github packages model instead of the default tag/release for when you update one of the formula in the tap?

very impressive homebrew tap BTW. 👍

**Comment by davidchall** [comment]
Hi @ipatch - thanks for the feedback!

I don't have a particular reason for using GitHub Packages with a homebrew tap. I just saw it was a possibility and wanted to try out this feature of GitHub. But I do think it is a nice way of organizing the bottles (you can easily list a history of versions for each package with their download statistics).

Are you aware of any other advantages/disadvantages?

**Reply by ipatch** [reply]
> Are you aware of any other advantages/disadvantages?

none that im aware of. with your current setup it looks like you get some analytics data which nice, but the default homebrew tap ci setup you don't really get any analytics to see which formulae people are using.

i might invest some time and move the homebrew-freecad tap to the package structure you're using. it also looks like the main homebrew-core tap uses the package structure as well. 👍
